content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,183 @@
+
+CCM displays costs for resources including query-level analysis, storage, and data transfer on the [**BigQuery dashboard**][1].
+
+## BigQuery pricing models
+
+BigQuery offers multiple pricing components, with CCM focusing on query-related processing costs.
+
+### Query Processing
+
+**On-demand queries**: You pay per query based on the amount of data processed.
+- Costs are directly attributed to individual queries based on bytes processed
+- Includes query-level tags for detailed cost attribution
+
+**Reservation-based queries**: You purchase dedicated processing capacity (slots) in advance at a fixed cost. Multiple queries can share this reserved capacity, making cost attribution more complex but potentially more cost-effective for consistent workloads.
+- Costs of reserved slots are allocated proportionally to queries using those slots
+- Allocation based on slot consumption (`total_slot_ms`) per query
+- Includes idle cost calculation for unused reservation capacity
+
+**Other BigQuery Costs:**
+- **Storage**: Charges for data stored in BigQuery tables (active and long-term storage)
+- **Streaming**: Costs for real-time data ingestion via streaming inserts
+- **Data Transfer**: Charges for moving data between regions or exporting data
+- **BI Engine**: Costs for in-memory analytics acceleration
+- **Other services**: ML training, routine executions, and additional BigQuery features
+
+CCM allocates and enriches costs for both query-processing pricing models, providing detailed cost attribution and tagging for your BigQuery analysis workloads. Learn more about BigQuery services and pricing models [**here**][3].
+
+[**Learn more about optimizing BigQuery performance and costs.**][8]
+
+## Prerequisites
+
+The following table presents the list of collected features and the minimal requirements:
+
+| Feature | Requirements |
+|---|---|
+| Retrieve tags from labels of a query | GCP CCM costs must be setup. Supported without monitoring or reservations. |
+| Query-Level Cost Attribution | BigQuery monitoring enabled |
+| Reservation Cost Allocation | BigQuery reservations configured |
+
+1. Configure the Google Cloud Cost Management integration on the [Cloud Cost Setup page][2].
+2. Enable BigQuery monitoring in your Google Cloud project. 
+[**Enable BigQuery monitoring here**][4]
+3. For reservation cost allocation, configure BigQuery reservations in your project. [**Learn about BigQuery reservations.**][7]
+
+## Allocating costs
+
+### Compute
+
+Costs are allocated into the following spend types:
+
+| Spend type | Description |
+|---|---|
+| `allocated_spend_type`: Usage | Cost of query execution based on bytes processed (on-demand) or slot consumption (reservation) |
+| `allocated_spend_type`: Cluster_idle | Cost of reserved slots allocated within a project but not utilized by queries|
```

Comment:
```suggestion
| `allocated_spend_type:usage` | Cost of query execution based on bytes processed (on-demand) or slot consumption (reservation) |
| `allocated_spend_type:cluster_idle` | Cost of reserved slots allocated within a project but not utilized by queries|
```

I think this is a bit cleaner and easier for a user to copy into their query editor, what do you think?